cityofaustin/atd-moped
# Combine support menu with avatar dropdown menu

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/10595

and some snackoos from https://github.com/cityofaustin/atd-data-tech/issues/10339:
- https://github.com/cityofaustin/atd-data-tech/issues/10604
- https://github.com/cityofaustin/atd-data-tech/issues/10605
- https://github.com/cityofaustin/atd-data-tech/issues/10603

## Testing
**URL to test:** 
https://deploy-preview-842--atd-moped-main.netlify.app/moped/projects

**Steps to test:**

1. use the new navigation menu (hamburger in corner) to confirm that the external links work, and the account link and logout button work
2. Check a project summary page and see that the cursor becomes a glove when hovering over the title. 
3. Add a ton of milestones to a milestones table, you shouldnt see any pagination and no WSOD

---
#### Ship list
